.circleci/soltest_all.sh

```diff
@@ -57,7 +57,7 @@ echo "Running steps $RUN_STEPS..."
 
 STEP=1
 
-[[ " $RUN_STEPS " =~ " $STEP " ]] && EVM=istanbul OPTIMIZE=1 ABI_ENCODER_V2=1 "${REPODIR}/.circleci/soltest.sh"
+[[ " $RUN_STEPS " =~ " $STEP " ]] && EVM=istanbul OPTIMIZE=1 ABI_ENCODER_V2=1 SOLTEST_ARGS="--no-smt" "${REPODIR}/.circleci/soltest.sh"
```

Comment:
Perhaps it is time to introduce LATEST_EVM on top?

Reply:
This also disables the smt optimizer tests, doesn't it? I would feel safer by using the `-t` option with negated patterns, if that is possible. Or introduce another `@`-tag.

Reply:
Different PR? :p

Reply:
@chriseth ok!